src/AzureAuth/AuthModeHelper.cs

```diff
@@ -0,0 +1,54 @@
+using Microsoft.Authentication.AzureAuth.Commands;
+using Microsoft.Authentication.MSALWrapper;
+using Microsoft.Extensions.Logging;
+using Microsoft.Office.Lasso.Interfaces;
+using Microsoft.Office.Lasso.Telemetry;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Authentication.AzureAuth
+{
+    /// <summary>
+    /// Helper class for <see cref="AuthMode"/>s.
+    /// </summary>
+    public static class AuthModeHelper
+    {
+        /// <summary>
+        /// Get the auth modes from the environment or set the default.
+        /// </summary>
+        /// <param name="env">The <see cref="IEnv"/> to use.</param>
+        /// <param name="eventData">Event data to add the auth mode to.</param>
+        /// <param name="logger">The <see cref="ILogger"/> to use.</param>
+        /// <returns>AuthModes.</returns>
+        public static IEnumerable<AuthMode> ReadAuthModeFromEnvOrSetDefault(IEnv env, EventData eventData, ILogger logger)

```

Comment:
Rather than creating a helper and adding a single method here, should we consider adding this to IEnvExtensions instead?

Helper classes always don't fit quite right in OOP, because they don't really represent an entity.

Reply:
I was debating between these two options, I will modify it :)